samigo/samigo-app/src/webapp/jsf/author/item/itemHeadings.jsp

```diff
@@ -14,6 +14,13 @@ for (i=0; i<document.links.length; i++) {
 document.links[newindex].onclick();
 }
 
+jQuery(document).ready(function($){
```

Comment:
What if jQuery.noConflict is used earlier in some JS? Would that mess up the dollar JSP expression?